node/actors/bft/src/leader/state_machine.rs

```diff
@@ -93,4 +93,104 @@ impl StateMachine {
         metrics::METRICS.leader_processing_latency[&label].observe_latency(ctx.now() - now);
         Ok(())
     }
+
+    /// In a loop, receives a PrepareQC and sends a LeaderPrepare containing it.
+    /// Every subsequent PrepareQC has to be for a higher view than the previous one (otherwise it
```

Comment:
Is it "higher" or "higher or equal"? Unless I'm missing something, the code below implements "higher or equal" check (i.e., the same view would pass the check). Not sure whether this matters, but it's at least a documentation nit 🙃

Reply:
Thanks for catching that. It matters, although not that much - it doesn't affect correctness, and we don't have slashing so nothing bad happens. At worst it would just send out useless messages.